res/config/config.sample.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+const BUGCACHE = [
+    "mysql" => "host=mysql:3306;user=bugcache;pass=bugcache;db=bugcache",
+    "redis" => "tcp://redis:6379",
+    /* https://developers.google.com/recaptcha/docs/faq */
+    "recaptchaKey" => "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI",
+    "recaptchaSecret" => "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe",
+];
+
+// Wait for MySQL to start up
+sleep(5);
+
+echo `mysql --host mysql -uroot -proot < /app/res/database/schema.sql`;
+echo `mysql --host mysql -uroot -proot bugcache < /app/res/database/init.sql`;
```

Comment:
I don't feel like you should put this into the config.sample.php file ... put it in a separate file to be used with docker only